unit_tests/actuator/CMakeLists.txt

```diff
@@ -14,4 +14,9 @@ if(ENABLE_OPENFAST)
    	  ${CMAKE_CURRENT_SOURCE_DIR}/UnitTestActuatorFunctorsFAST.C
    	  ${CMAKE_CURRENT_SOURCE_DIR}/UnitTestActuatorBulkDiskFAST.C
    )
+   # create symlink to nrelmw.fst 
+   execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink
+     ${CMAKE_BINARY_DIR}/reg_tests/test_files/nrel5MWactuatorLine/nrel5mw.fst
```

Comment:
Any specific reason to not use `${CMAKE_SOURCE_DIR}` here?

Reply:
@sayerhs in `${CMAKE_SOURCE_DIR}` `nrel5mw.fst` is a `nrel5mw.fst.in` due to cmake variables that get populated during the build process.